test/gpu/miopen.cpp

```diff
@@ -1115,6 +1233,14 @@ int main()
     verify_program<test_add_broadcast4>();
     verify_program<test_add_broadcast5>();
     verify_program<test_triadd_broadcast>();
+    verify_program<test_sub>();
+    verify_program<test_sub2>();
+    verify_program<test_sub_broadcast>();
+    verify_program<test_sub_broadcast2>();
+    verify_program<test_sub_broadcast3>();
+    verify_program<test_sub_broadcast4>();
+    verify_program<test_sub_broadcast5>();
+    verify_program<test_sub_broadcast6>();
```

Comment:
I dont think the broadcast test are necessary.

Reply:
fixed